src/api/app/policies/attrib_policy.rb

```diff
@@ -3,11 +3,14 @@ def create?
     # Admins can write everything
     return true if @user.is_admin?
 
-    # No specific rules set for the attribute, check if the user can modify the container
-    return true if @record.container.try(:can_be_modified_by?, @user)
-
-    # check for modifiable_by rules
-    permissions_for_modifiables?(modifiables)
+    at_modifiables = modifiables
```

Comment:
Only a suggestion - up to you if you want to implement it or not but you can delete this line if you change memoize the `modifiables` method like this:
```ruby
  def modifiables
    @modifiables ||=
      @record.attrib_type.try(:attrib_type_modifiable_bies).to_a |
        @record.attrib_type.try(:attrib_namespace).try(:attrib_namespace_modifiable_bies).to_a
  end
```